packages/integration-sdk-runtime/src/execution/executeIntegration.ts

```diff
@@ -62,7 +63,7 @@ export interface ExecuteIntegrationOptions {
   enableSchemaValidation?: boolean;
   graphObjectStore?: GraphObjectStore;
   createStepGraphObjectDataUploader?: CreateStepGraphObjectDataUploaderFunction;
-  resultsCallback?: (results: IntegrationStepResult[]) => Promise<void>;
+  resultsCallback?: (results: ExecuteIntegrationResult) => Promise<void>;
```

Comment:
I should've passed the `ExecuteIntegrationResult` anyway from the start tbh. I don't usually use lists as the parameter for a public interface. My bad.

Reply:
I am updating the runtime to allow any to be sent to storeJobOutputJsonInS3 since it should be able to take anything serializable. Now that you mention it that would require the 2 to be updated together. (or at least upgrading the sdk would require the managed sdk to be updated first)